server/accounts.go

```diff
@@ -2868,7 +2895,8 @@ func (s *Server) updateAccountClaimsWithRefresh(a *Account, ac *jwt.AccountClaim
 		switch e.Type {
 		case jwt.Stream:
 			s.Debugf("Adding stream export %q for %s", e.Subject, a.Name)
-			if err := a.AddStreamExport(string(e.Subject), authAccounts(e.TokenReq)); err != nil {
+			if err := a.AddStreamExportWithAccountPos(
```

Comment:
Would change to private if you agree that it does not have to be exported at this point.

Reply:
fixed